DEVELOPER.md

```diff
@@ -0,0 +1,45 @@
+# Guide for developers
+This project is a plugin working in docker engine for delivering log to Sumo Logic cloud service through HTTP source.
+This is the guide for developers want build and extend the plugin. If you just want use this plugin in your docker environment, please refer to the [readme file](README.md).
+
+## Prerequisite
+  * [Download](https://www.docker.com/get-docker) and install latest docker engine
+  * [Download](https://golang.org/dl/) and install latest Go language distribution
+  * Clone/Download this repository to a local directory, and
+  * Get all dependencies with 
+  ```bash
+  $ go get -d ./...
+  ```
+
+## Build and install plugin to docker
+In bash, run:
+```bash
+$ sudo bash ./plugin_install.sh
+```
+If everything goes fine, you can verify the plugin correctly installed with:
```

Comment:
`If everything goes fine, you can verify that the plugin is correctly installed with:`